src/components/Page-Title/Page-Title.js

```diff
@@ -0,0 +1,73 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class PageTitle extends React.Component {
+  render() {
+    let help = '';
+    let link = '';
+    let subHeader = '';
+
+    if (this.props.helpIcon) {
+      help = (
+        <a
+          className="chi-button -icon -flat -lg -bg--none -b--transparent -opacity-hover--80"
+          onClick={this.props.helpclick}
+          onMouseEnter={this.props.mouseoverhelp}
+          onMouseLeave={this.props.mouseleavehelp}>
+          <div className="chi-button__content">
+            <i className="chi-icon icon-circle-question-outline"></i>
+          </div>
+        </a>
+      );
+    }
+
+    if (this.props.backlink) {
+      link = (
+        <a
+          className="chi-link"
+          onClick={this.props.linkclick}>
+          <div className="chi-link__content">
+            <i className="chi-icon icon-chevron-left -xs"></i>
+            <span className="-text--md">{this.props.backlink ? this.props.backlink : ''}</span>
+          </div>
+        </a>
+      );
+    }
+
+    if (this.props.subtitle) {
+      subHeader = (
```

Comment:
The naming is a bit inconsistent (subTitle and subHeader)

Reply:
changed to subtitle